lib/min_heap.rb

```diff
@@ -14,18 +14,23 @@ def initialize
   end
 
   # This method adds a HeapNode instance to the heap
-  # Time Complexity: ?
-  # Space Complexity: ?
+  # Time Complexity: O(n(log(n))) where n is the number of nodes in the heap
```

Comment:
Since you are making at most `log n` swaps to add something to a heap, the time complexity is O(log n)